hal/src/nRF52840/spi_hal.cpp

```diff
@@ -299,6 +299,9 @@ void HAL_SPI_Begin_Ext(HAL_SPI_Interface spi, SPI_Mode mode, uint16_t pin, void*
             // m_spi_map[spi].ss_pin = PIN_INVALID;
         }
     } else {
+        if (pin >= TOTAL_PINS) {
```

Comment:
This seams to be resided in another PR. Should it be included in this PR?

Reply:
:+1: I'll remove it.

Reply:
Rebased and removed.